include/audiomgr.h

```diff
@@ -0,0 +1,33 @@
+#ifndef AUDIOMGR_H
+#define AUDIOMGR_H
+
+#include "sched.h"
+#include "z64audio.h"
+
+typedef enum {
+    AUDIOMGR_DEBUG_LEVEL_NONE = 0,
+    AUDIOMGR_DEBUG_LEVEL_NO_RSP = 1,
+    AUDIOMGR_DEBUG_LEVEL_NO_UPDATE = 2
+} AudioMgrDebugLevel;
```

Comment:
we can drop the value assignments right?
theoretically would want offset comments to stay true to the repos convention, but I personally do not care much for them in this case, but meh